src/tests/execution/skippingStepTest.ts

```diff
@@ -0,0 +1,7 @@
+import { Feature } from '../..';
+
+Feature('./execution.feature', ({ Scenario }) => {
+  Scenario('Steps are skipped when their function is omitted')
+    .Then('this test should be skipped')
+    .And.skip('this test should also be skipped');
```

Comment:
- The `.And` step should be skipped whether you add `.skip` or not, as the previous step was already skipped
- Ideally, you want a second scenario to show that using `.skip` also skips consecutive steps

Reply:
Great point, I started off just testing the compilation.